libs/ui-lib-tests/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "resolveJsonModule": true,
     "types": ["cypress", "node", "@testing-library/cypress"]
   },
-  "include": ["cypress.config.ts", "cypress/**/*.ts"]
-}
+  "include": ["cypress.config.ts", "cypress/**/*.ts", "../../node_modules/cypress-file-upload"]
```

Comment:
This is not the why it's done, PTAL here: https://www.npmjs.com/package/cypress-file-upload#installation

Reply:
True, I've changed it